modules/administration-guide/partials/proc_installing-image-puller-using-che-operator.adoc

```diff
@@ -15,9 +15,10 @@ include::example$snip_{project-context}-installing-image-puller-using-che-operat
 
 .Procedure
 
-. Edit the `CheCluster` Custom Resource and set `.spec.imagePuller.enable` to `true`
+. Edit the `CheCluster` Custom Resource and set `.spec.imagePuller.enable` to `true`:
++
+.. Enable {image-puller-name-short} in the `CheCluster` Custom Resource:
```

Comment:
Sorry, spoke to soon, could we combine these two lines (line 18 and 20) to something like:
```
. Enable {image-puller-name-short} in the `CheCluster` Custom Resource by setting `.spec.imagePuller.enable` to `true`:
```

Reply:
Yeah, I was thinking about it too.